index.html

```diff
@@ -0,0 +1,97 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Covered+By+Your+Grace&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <div class="brand">
+      <h1 class="brand-name">Taiba Hijab Center</h1>
+      <p class="discription">THANE'S PREMIUM HIJAB CENTER</p>
+    </div> 
+    <input type="checkbox" id="openside">
+    <label for="openside" class="sidebaricon">
+      <div class="spinner top"></div>
+      <div class="spinner mid"></div>
+      <div class="spinner bottom"></div>
+    </label>
+    <nav class="sidebar">
+      <ul class="menu">
+        <li><a href="#">NEW ARRIVALS</a></li>
+        <li><a href="#">HIJAB</a></li>
+        <li><a href="#">DRESESS</a></li>
+        <li><a href="#">ACCESSORIES</a></li>
+        <li><a href="#">ABOUT</a></li>
+      </ul>
+      <ul class="social-network">
+        <li><a href="#" class="insta"><img src="./images/1.png" alt="insta"></a></li>
+        <li><a href="#" class="fb"><img src="./images/2.png" alt="facebook"></a></li>
+      </ul>
+    </nav>
+   </div>
+  </header>
+  <article class="slideshow">
+    <div class="slider">
+      <div class="slide one"><button>NEW ARRIVALS</button></div>
+      <div class="slide two"></div>
+      <div class="slide three"></div>
+      <div class="slide four"><button>SHOP NOW</button></div>
+    </div>
+  </article>
+  <section class="shop">
+      <div class="hijabp one">
+        <img src="./images/plain hijab1.jpg" alt="product">
+        <h3>Scheffon Fabric Hijab</h3>
+        <p> ₹599</p>
+        <button class="buy">BUY NOW</button>
+      </div>
+      <div class="hijabp two">
+        <img src="./images/plain hijab2.jpg" alt="product">
+        <h3>Scheffon Fabric Hijab</h3>
+        <p> ₹599</p>
+        <button class="buy">BUY NOW</button>
+      </div>
+      <div class="hijabp three">
+        <img src="./images/plain jiab 3.webp" alt="product">
+        <h3>Scheffon Fabric Hijab</h3>
+        <p> ₹599</p>
+        <button class="buy">BUY NOW</button>
+      </div>
+      <div class="hijabp four">
+        <img src="./images/GeogetHijab1.jpg" alt="product">
+        <h3>Scheffon Fabric Hijab</h3>
+        <p> ₹599</p>
+        <button class="buy">BUY NOW</button>
+      </div>
+  </section>
+  <section class="addv">
+      <h2 class="adiscp"> Buy Hijab Online India</h2>
+      <p>Buy Hijabs Online At The <span> Best Price</span>.Introduces <span>High-Quality Hijabs</span> In India,<br> <span>Specialising</span> In All The <span>Latest Hijabs</span><br> Such As Chiffon Hijab, Hijab Dupatta, Abaya Hijab, Hijab Dress, Black Hijab, Readymade Hijab, Wedding Hijab,<br> New Model Hijab Style. <br> You Can Purchase Our Hijabs Online Using <span> Cash On Delivery</span>,<span>Net Banking</span>, <br><span>Credit Card</span>,<span> Debit Card</span>.</p>
```

Comment:
Try using class inside span. So that While styling only the span with specific class eg: pink/maroon will get styled. And you can also use multiple p tags instead of line breaks, the code will look more cleaner

Reply:
just give all of them clasess of .purplle